mne/connectivity/spectral.py

```diff
@@ -469,6 +469,8 @@ def _get_n_epochs(epochs, n):
         if len(epochs_out) >= n:
             yield epochs_out
             epochs_out = list()
+    if 0 < len(epochs_out) < n:
```

Comment:
Can you also include case ``len(epochs_out) == 0``? It causes some joblib versions to hang forever. Fixed in https://github.com/mne-tools/mne-python/pull/4278, but apparently got lost somewhere.